Changes

```diff
@@ -8,16 +8,18 @@ Next version (4.04.1):
 
 ### Bug fixes
 
-- PR#7405, GPR#903: s390x: Fix address of caml_raise_exn in native dynlink modules
-  (Richard Jones, review by Xavier Leroy)
+- PR#7405, GPR#903: s390x: Fix address of caml_raise_exn in native
+  dynlink modules (Richard Jones, review by Xavier Leroy)
 
-- MPR#7417, GPR#930: ensure 16 byte stack alignment inside caml_allocN on x86-64
-  for ocaml build with WITH_FRAME_POINTERS defined
+- MPR#7417, GPR#930: ensure 16 byte stack alignment inside caml_allocN
+  on x86-64 for ocaml build with WITH_FRAME_POINTERS defined
   (Christoph Cullmann)
 
-- GPR#912: Fix segfault in Unix.create_process on Windows caused by wrong header
-  configuration.
-  (David Allsopp)
+- GPR#912: Fix segfault in Unix.create_process on Windows caused by
+  wrong header configuration.  (David Allsopp)
+
+ -GPR#980: add dynlink options to ocamlbytecomp.cmxa to allow ocamlopt.opt
```

Comment:
Oh, and the previous GPR#912 has been corrupted (again, I guess your editor) - the first line is exactly 80 columns and the credit should be on its own line.